src/components/member-card/index.js

```diff
@@ -7,7 +7,7 @@ import PropTypes from 'prop-types';
 import ShowSkills from '@components/member-card/show-skills';
 import SuperUserOptions from '@components/member-card/super-user-options';
 
-const Card = ({ developerInfo }) => {
+const Card = ({ developerInfo, optionKey }) => {
```

Comment:
I do not see `optionKey` added in the props list
https://github.com/Real-Dev-Squad/website-members/pull/303/files#diff-cf115ea7b3dbc387c34873a734a99c2166b7fb70acd94ccb24125d9d949e431dR85